piksi_tools/console/settings.yaml

```diff
@@ -404,6 +404,21 @@
     for each timestamp. When "No DGNSS" is chosen, no differential output will be
     attempted by the receiver.
 
+- group: solution
+  name: process_noise_motion_mode
+  expert: false
+  type: enum
+  units: N/A
+  default value: 'Default'
+  readonly: false
+  enumerated possible values: Default, Automotive, Static
```

Comment:
As for the other PR, I'm a bit unsure what our options should be. If we want to keep the current configuration with high process noise, we should probably just name it something that makes it obvious what it is (or perhaps "UAV", if that's what the UAV use case will use). It sounds like "Static" has changed a bit, but why does the static case have non-zero process noise? Is it assumed that tectonic motion needs to be considered? If it's more than that, I'd think the name should be changed to "Low dynamics" or something.